Remove redundant tests/mmap_file.c. NFC

sbc100 opened the pull request:
We already have tests/fs/test_mmap.c that covers mapping files,
including offsets.

Also tests/test_mmap.c to make to more clear what its testing.

tlively commented:
Why do we need to keep these tests around (albeit renamed) if there's also fs/test_mmap.c?

sbc100 commented:
> Why do we need to keep these tests around (albeit renamed) if there's also fs/test_mmap.c?

I'm not sure what you mean.. in one case I just renamed a test.. in the other I deleted it because it was redundant.. why would we want to keep it around?

sbc100 commented:
> > Why do we need to keep these tests around (albeit renamed) if there's also fs/test_mmap.c?
> 
> I'm not sure what you mean.. in one case I just renamed a test.. in the other I deleted it because it was redundant.. why would we want to keep it around?

If you are refering the browser test.. then no I don't think there is any reason to keep it around.    The old test was just for mmap from MEMFS which is not related to browser.   There are many many things we could run in browser tests but don't.. I don't see why this test should be an exception.
